.github/workflows/deploy-DEV.yml

```diff
@@ -63,6 +63,8 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
@vkucera You know that here is only for our CI, not the templates that are generated by the mega-linter-runner when a user first installs Megalinter. Is that what you wanted?

Reply:
@echoix I added it to pass the CI. Please see https://github.com/oxsecurity/megalinter/pull/3472#issuecomment-2042200621

Reply:
Ok good!